### DISCUSSION TITLE
The tea x pkgx Hackathon: Revolutionizing Smart Contract Code Review and Auditing

### DISCUSSION THREAD

**Discussion by michaelessiet** [pkgxdev/pkgx#936] [Show and Tell]
This discussion is for developers taking part in the hackathon to ask questions and share their progress.

**Comment by 0xiDen** [comment]
How can we take part in the hackathon?

**Reply by michaelessiet** [reply]
The hackathon is open for anyone to participate. Just make sure to submit your solution once it's time for submission. Submission will be announced in the tea [discord](https://discord.gg/tea-906608167901876256) as well as both the [pkgx](https://twitter.com/pkgxdev) and [tea](https://twitter.com/teaprotocol) twitter accounts.

**Reply by Fidex123** [reply]
When teatnet
